chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -1146,25 +1130,24 @@ private CertifiedProductSearchDetails validateCertifiedProduct(
      * @param file
      * @param ex
      */
-    private void sendUploadError(MultipartFile file, Exception ex) {
+    private void sendUploadError(final MultipartFile file, final Exception ex) {
         //get the recipients of this email
         //if there are none specified we won't continue
-        String[] toProperty = env.getProperty("uploadErrorEmailRecipients").split(",");
-        if(toProperty == null || toProperty.length == 0) {
+        if (StringUtils.isEmpty(env.getProperty("uploadErrorEmailRecipients"))) {
             return;
         }
-        List<String> recipients = Arrays.asList(toProperty);
+        List<String> recipients = Arrays.asList(env.getProperty("uploadErrorEmailRecipients").split(","));
 
         //figure out the filename for the attachment
         String originalFilename = file.getOriginalFilename();
         int indexOfExtension = originalFilename.indexOf(".");
         String filenameWithoutExtension = file.getOriginalFilename();
-        if(indexOfExtension >= 0) {
-            filenameWithoutExtension = 
-                    originalFilename.substring(0, indexOfExtension);
+        if (indexOfExtension >= 0) {
+            filenameWithoutExtension
+            = originalFilename.substring(0, indexOfExtension);
```

Comment:
Should this be indented?  https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent